.docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:22.04
```

Comment:
🗒️ This is the Dockerfile that defines a container running an sshd server. We use it as the deployment target for the Cucumber tests.